hermes-main-website/src/app/components/header/header.component.sass

```diff
@@ -0,0 +1,87 @@
+@import 'src/variables.sass'
+
+.content-menu li
+    display: inline-block
+    background-color: $red-main
+    font-size: 25px
+    margin:0
```

Comment:
Delete the margin:0 

And not a mistake but please update font-size here to 20px :D thanks :D